src/clarity-angular/popover/dropdown/dropdown-toggle.ts

```diff
@@ -30,7 +30,7 @@ export class DropdownToggle {
     }
 
     get active(): boolean {
-        return this._dropdown.open;
+        return this._dropdown.ifOpenService.open;
     }
 
     @HostListener("click")
```

Comment:
@jeeyun: I walk talking about the click listener. I couldn't comment on the exact line because you didn't modify it. 😉